index.html

```diff
@@ -1292,10 +1292,11 @@
 
 ### Ancillary uses
 
-[=Sites=] sometimes use data in ways that aren't needed for the user's immediate
+[=Sites=] sometimes use data in ways that aren't needed for the user's primary
 goals. For example, they might bill advertisers, measure site performance, or
-tell developers about bugs. These uses are known as <dfn data-lt="ancillary
-use">ancillary uses</dfn>.
+tell developers about bugs. These uses, which provide indirect support for,
+but are not inherent in, the user's primary goals, are known as
+<dfn data-lt="ancillary use">ancillary uses</dfn>.
 
```

Comment:
FWIW, I debated between “inherent in” and “fundamental to” here. The former seems more accurate to the point, the second reads a little more comfortably to me. I suspect either one could work.